decidim-conferences/app/views/decidim/conferences/admin/media_links/index.html.erb

```diff
@@ -8,37 +8,56 @@
       <% end %>
     </h1>
   </div>
-  <div class="table-scroll">
+  <div class="table-stacked">
     <table class="table-list">
       <thead>
         <tr>
           <th><%= t("models.media_link.fields.title", scope: "decidim.admin") %></th>
           <th><%= t("models.media_link.fields.link", scope: "decidim.admin") %></th>
           <th><%= t("models.media_link.fields.date", scope: "decidim.admin") %></th>
-
-          <th></th>
+          <th><%= t("models.media_link.fields.actions", scope: "decidim.admin") %></th>
         </tr>
       </thead>
       <tbody>
         <% @media_links.each do |media_link| %>
           <tr>
-            <td>
+            <td data-label="<%= t("models.media_link.fields.title", scope: "decidim.admin") %>">
               <%= translated_attribute(media_link.title) %>
             </td>
-            <td>
+            <td data-label="<%= t("models.media_link.fields.link", scope: "decidim.admin") %>">
               <%= media_link.link %>
             </td>
-            <td>
+            <td data-label="<%= t("models.media_link.fields.date", scope: "decidim.admin") %>">
               <%= l(media_link.date, format: :decidim_short ) %>
             </td>
-            <td class="table-list__actions">
-              <% if allowed_to? :update, :media_link, media_link: media_link %>
-                <%= icon_link_to "pencil-line", edit_conference_media_link_path(current_conference, media_link), t("actions.edit", scope: "decidim.admin"), class: "action-icon--edit" %>
-              <% end %>
+            <td class="table-list__actions" data-label="<%= t("models.media_link.fields.actions", scope: "decidim.admin") %>">
+              <button type="button" data-component="dropdown" data-target="actions-media-link-<%= media_link.id %>" aria-label="<%= t("decidim.admin.actions.actions_label", resource: translated_attribute(media_link.title)) %>">
+                <%= icon "more-fill", class: "text-secondary" %>
+              </button>
+
+              <div class="inline-block relative">
+                <ul id="actions-media-link-<%= media_link.id %>" class="dropdown dropdown__action" aria-hidden="true">
+                  <% if allowed_to? :update, :media_link, media_link: media_link %>
+                    <li class="dropdown__item">
+                      <%= link_to edit_conference_media_link_path(current_conference, media_link), class: "dropdown__button" do %>
+                        <%= icon "pencil-line" %>
+                        <%= t("actions.edit", scope: "decidim.admin") %>
+                      <% end %>
+                    </li>
+                  <% end %>
+
+                  <hr>
```

Comment:
Let's move the HR inside the above permission check

Reply:
Fixed on 2edcdcb5ea1d34de68fb35c5d3038169988dc759